packages/ocean-react/src/CrossSellCard/types.ts

```diff
@@ -0,0 +1,37 @@
+import { ComponentPropsWithoutRef } from 'react';
```

Comment:
It's an ocean-web pattern to leave the types in the same file that they are used to easy out the maintenance and understanding of the component attributes.